plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -259,20 +270,39 @@ public Optional<TrinoPrincipal> getSchemaOwner(ConnectorSession session, Catalog
     @Override
     public IcebergTableHandle getTableHandle(ConnectorSession session, SchemaTableName tableName)
     {
+        return getTableHandle(session, tableName, Optional.empty(), Optional.empty());
```

Comment:
I think we shouldn't need to implement both overloads: posted https://github.com/trinodb/trino/pull/12542 with a proposed changed.

For now -- no change requested here.